core/storage/suggestion/gae_models_test.py

```diff
@@ -225,3 +225,9 @@ def test_get_suggestions_by_target_id(self):
                 .get_suggestions_by_target_id(
                     suggestion_models.TARGET_TYPE_EXPLORATION, 'exp_invalid')),
             0)
+
+    def test_get_all_stale_suggestions(self):
+        with self.swap(suggestion_models, 'THRESHOLD_TIME_BEFORE_ACCEPT', 0):
```

Comment:
Um we can do better. When we say 0, we are actually not testing the logic which takes a time delta and subtracts it from the current time. It is also fine if the delay is a few msecs only.

Reply:
so you mean put about 10 msec or something like that instead of 0?

Reply:
I added a separate test that checks with the actual value and returns an empty list. This makes sure that the subtraction logic works fine. Is this fine?

Reply:
Well, unless you are asserting the actual time calculated, you won't know if it is correct or not. One way to verify could be to retrieve a suggestion just before that calculated time or you could abstract that functionality into a method and test that method separately.